### ISSUE TITLE
Make no available subtitles (or captions) more obvious

### ISSUE THREAD

**Issue by jmbreuer** [TeamNewPipe/NewPipe#6339]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [X] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [X] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

The closed captions / subtitles control should still be visible and disabled or slow only a "no subtitles [available]" entry, when there are no subtitles available for a specific video from a provider that supports subtitles in principle (ie YouTube). 

#### Is your feature request related to a problem? Please describe it

I started to watch a video in a foreign language and searched all of NewPipe's UI high and low for how to turn on subtitles before I realized this particular video doesn't have any and this causes NewPipe to completely remove the subtitle controls. 

#### Additional context

This video: https://youtu.be/DcWhtTXaWE4 

#### How will you/everyone benefit from this feature?

New users will be less likely to think that NewPipe may be missing subtitle functionality in general, which is what happened to me at first.

Less confusion.

**Comment by SameenAhnaf** [comment]
@mhmdanas Isn't this a duplicate of #5378?

**Comment by mhmdanas** [comment]
No, it's not. This issue requests that the caption control in the player remain even when there are no captions and that it should show that no captions are available, while that issue requests for a CC/subtitle thingy which can be seen before even opening the video. Somewhat similar, but not quite the same.
